local-cluster/tests/common/mod.rs

```diff
@@ -478,20 +478,12 @@ impl SnapshotValidatorConfig {
         accounts_hash_interval_slots: Slot,
         num_account_paths: usize,
     ) -> SnapshotValidatorConfig {
+        // Interval values must be nonzero
         assert!(accounts_hash_interval_slots > 0);
         assert!(full_snapshot_archive_interval_slots > 0);
+        assert!(incremental_snapshot_archive_interval_slots > 0);
```

Comment:
For consistency between `solana-validator` and local-cluster tests (or really any other callers), we could move the checks these assertions perform into `is_snapshot_config()` valid.

I don't feel too strongly on that point - I care more about replacing the stuff I'm ripping out with the helper